### DISCUSSION TITLE
problème fermeture pop up

### DISCUSSION THREAD

**Discussion by gorgli** [AmauriC/tarteaucitron.js#954] [General]
bonjour,
j'ai un petit problème sur un site ou j'utilise tarte au citron dans un grande pop up, 
lorsque le visiteur accepte les cookies la pop up de tac ce ferme mais ferme également les pop up qui sont présentent derrière comment régler ce problème et garder les pop up derrière active svp ?

**Comment by AmauriC** [comment]
Possible d'avoir l'url du site? Les autres popups sont en dehors de tarteaucitron?

**Reply by gorgli** [reply]
le pop up de fond est géré via un autre plugin en dehors de tarte au citron

**Reply by AmauriC** [reply]
Je viens de tester, une fois ça a été masqué, maintenant ça semble ok. Un changement entre temps? :D

**Reply by gorgli** [reply]
oui j'ai trouvé une solution merci quand même pour la rapidité de réponse

**Reply by gorgli** [reply]
la pop up de fond avait un réglage qui permet de la fermer en cliquant n'importe ou sur l'écran y compris le bouton de la pop up  tac du coup ^^
